translate3_0to2_2.go

```diff
@@ -320,7 +321,12 @@ func translateFiles3to2(files []types.File, fss []string) (ret []old.File) {
 		// Overwrite defaults to false in spec 3 and true in spec 2; but
 		// we should omit it if append is specified as we want the "unset"
 		// default.
-		if f.Node.Overwrite == nil && len(f.FileEmbedded1.Append) == 0 {
+		if f.Node.Overwrite != nil {
+			file.Node.Overwrite = f.Node.Overwrite
+		}
+
+		// If append is true overwrite must be false in spec 3
+		if f.FileEmbedded1.Append != nil {
```

Comment:
@bgilbert note this change in logic here (I'll split that into a separate commit)

Reply:
this is in a separate commit now.